### PR TITLE
Fix issue 2207: fatal error when canceling actions

### PR DESCRIPTION
**Pull request by gggeek**

### CODE REVIEW

File: lib/Drush/Log/Logger.php [hunk 1]
```diff
@@ -27,13 +27,30 @@
...
 use  Symfony\Component\Console\Output\OutputInterface;
 
-class Logger extends RoboLogger {
+class Logger extends BaseLogger {
 
     public function __construct(OutputInterface $output) {
-      parent::__construct($output);
+
+      $verbosityOverrides = [
+        // The first 3 lines are kept for backwards compatibility, as thsi class was extending RoboLogger beforehand
```
Comment (andrerom) [inline lib/Drush/Log/Logger.php]: thsi => this

;)

Comment (gggeek) [inline lib/Drush/Log/Logger.php]: geez, those nitpickers! :-D

Comment (andrerom) [inline lib/Drush/Log/Logger.php]: hence the joke :)

**Comment by weitzman** [comment]
Can you please provide steps to reproduce the fatal?

**Pull request closed without merging**